README.md

```diff
@@ -56,38 +56,21 @@ Particular.PlatformLauncher.Launch(servicePulseDefaultRoute: "/monitoring");
 
 ## Updating
 
-For now, the platform binaries (or in the case of ServicePulse, web assets) are embedded directly in this repository and must be updated manually when new versions of the platform tools are released.
-
-### Updating ServiceControl & Monitoring
-
-1. In TeamCity, find the latest ServiceControl build on the `master` branch, download the following artifacts, and unzip:
-    * zip/Particular.ServiceControl-{VERSION}.zip
-    * zip/Particular.Servicecontrol.Monitoring-{VERSION}.zip
-2. Update the ServiceControl binaries:
-    1. In the repository, delete the contents of `src\Particular.PlatformSample\platform\servicecontrol\servicecontrol-instance`.
-    2. Replace with the contents of the `ServiceControl` folder from the ServiceControl zip file.
-    3. Add the contents of the `Transports/LearningTransport` folder from the ServiceControl zip file.
-3. Update the Monitoring binaries:
-    1. In the repository, delete the contents of  `src\Particular.PlatformSample\platform\servicecontrol\monitoring-instance`.
-    2. Replace with the contents of the `ServiceControl.Monitoring` folder from the Monitoring zip file.
-    3. Add the contents of the `Transports/LearningTransport` folder from the Monitoring zip file. This will include duplicates (things like Autofac, Nancy, etc.) that you can just not replace in the destination directory. Also skip 
-4. Check the new configuration files for structural changes:
-    * The config files in [src/Particular.PlatformSample/configs](https://github.com/Particular/Particular.PlatformSample/tree/master/src/Particular.PlatformSample/configs) are embedded resources that use simple string replacement to replace values such as `{ServiceControlPort}` with the correct values.
-    * Compare the **ServiceControl.exe.config** and **Servicecontrol.Monitoring.exe.config** files to the newly updated sources ensuring the structure is the same, given that `TransportType` will be configured for the Learning Transport. Consult with ServiceControl maintainers if necessary.
-5. Remove any pdb files or config files that show up as file additions in the git diff.
+ServiceControl, ServiceControl.Monitoring, and ServicePulse are all NuGet packaged dependencies of this project. Each of these publishes a PlatformSample specific NuGet with each release:
```

Comment:
Grammar nitpick, should either be "NuGet-packaged dependencies" or "NuGet package dependencies"

Reply:
```suggestion
ServiceControl, ServiceControl.Monitoring, and ServicePulse are all NuGet package dependencies of this project. Each of these publishes a PlatformSample specific NuGet with each release:
```